nakedret.go

```diff
@@ -211,3 +194,76 @@ func (v *returnsVisitor) Visit(node ast.Node) ast.Visitor {
 
 	return v
 }
+
+func checkRequestedFiles(dirName string, maxLength uint, shouldFix bool) {
+	path, err := filepath.Abs(dirName)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	checkAllDirectories(path, maxLength, shouldFix)
+}
+
+func checkAllDirectories(path string, maxLength uint, shouldFix bool) {
```

Comment:
Shouln't we be bubbling these errors up? Currently, it looks like this will silently fail in some cases.

Reply:
I chose to log errors in this func as a means of alerting rather than bubbling up. But there is a few opportunities for improvement.